src/events/ready/02send-message-roles.js

```diff
@@ -5,6 +5,7 @@ const {ActionRowBuilder, ButtonStyle, ButtonBuilder} = require('discord.js');
 //send the roles message into a specific channel - id is inside of config.json
 module.exports = async(client) =>{
     try {
+        return;
```

Comment:
Can this file be deleted, or are we using this later?

Reply:
I suppose it can be deleted. There is no reason to keep it as of right now.